tests/phpunit/Unit/Outline/OutlineResultPrinterTest.php

```diff
@@ -82,9 +82,37 @@ public function testGetResult_LinkOnNonFileOutput() {
 			->method( 'getValue' )
 			->will( $this->returnValue( '' ) );
 
+		$introtemplate = $this->getMockBuilder( '\stdClass' )
+			->disableOriginalConstructor()
+			->setMethods( [ 'getName', 'getValue' ] )
+			->getMock();
+
+		$introtemplate->expects( $this->any() )
+			->method( 'getName' )
+			->will( $this->returnValue( 'introtemplate' ) );
+
+		$introtemplate->expects( $this->any() )
+			->method( 'getValue' )
+			->will( $this->returnValue( '' ) );
+
+		$outrotemplate = $this->getMockBuilder( '\stdClass' )
+			->disableOriginalConstructor()
+			->setMethods( [ 'getName', 'getValue' ] )
+			->getMock();
+
+		$outrotemplate->expects( $this->any() )
+			->method( 'getName' )
+			->will( $this->returnValue( 'outrotemplate' ) );
+
+		$outrotemplate->expects( $this->any() )
+			->method( 'getValue' )
+			->will( $this->returnValue( '' ) );
+
```

Comment:
Oh man this is an unfortunate pattern that you copied here. Thanks for writing the test though!